man/mono.1

```diff
@@ -1652,11 +1652,11 @@ Automatically generates sequence points where the
 IL stack is empty.  These are places where the debugger can set a
 breakpoint.
 .TP
-\fBgen-compact-seq-points\fR
```

Comment:
yes, it was shipped in a release. It was used in order to mono-symbolicate to work. We can keep it but it won't do anything. @kumpera what do you think, should we keep it?